awx/main/models/unified_jobs.py

```diff
@@ -291,16 +278,6 @@ def _get_current_status(self):
         else:
             return 'successful'
 
-    def _get_last_job_run(self):
-        # Override in subclasses as needed.
-        if self.last_job:
-            return self.last_job.finished
-
-    def _set_status_and_last_job_run(self, save=True):
-        status = self._get_current_status()
-        last_job_run = self._get_last_job_run()
-        return self.update_fields(status=status, last_job_run=last_job_run, save=save)
```

Comment:
@chrismeyersfsu asked the obvious question - how does the status get set? Quoting myself from the issue.

>  _update_parent_instance is a UnifiedJob method that is redundant with _set_status_and_last_job_run on the UnifiedJobTemplate model.